doc/nix-local-build.rst

```diff
@@ -170,8 +170,9 @@ identify the result of a build; if we compute this identifier and we
 find that we already have this ID built, we can just use the already
 built version.
 
-The global package store is ``~/.cabal/store`` (configurable via
-global `store-dir` option); if you need to clear your store for
+The global package store is typically ``~/.local/state/cabal`` on Unix and
```

Comment:
Both should be listed, since for backward compatibility `~/.cabal` will be used in preference to the XDG paths if it is present.

Reply:
Maybe a future proof “invoke `cabal path` to see where your local store is located”?

Reply:
Thanks for the suggestion, I've reworded.